project/Dependencies.scala

```diff
@@ -22,6 +22,8 @@ object Dependencies {
   val CouchbaseVersion = "2.7.2"
   val CouchbaseVersionForDocs = "2.7"
 
+  val JwtCoreVersion = "2.1.0"
```

Comment:
The 2.13 build failed because we upgraded from Scala 2.13.0-M5 to 2.13.0 in master and jwt-core 2.1.0 is not available for Scala 2.13.0. It is a bit tricky to upgrade jwt-core for libraries that are using it already (#1762 and #1763), but since this is a new connector, this can be upgraded to 3.0.1 safely.

Reply:
I updated JwtCoreVersion to `3.0.1` together with failing code.

I noticed these warnings when compiling against Scala `2.13.0`

```
[warn] /Users/jgrgurica/Development/FOOS/alpakka/google-cloud-storage/src/main/scala/akka/stream/alpakka/googlecloud/storage/FailedUpload.scala:15:47: object JavaConverters in package collection is deprecated (since 2.13.0): Use `scala.jdk.CollectionConverters` instead
[warn]   def getReasons: java.util.List[Throwable] = reasons.asJava
[warn]                                               ^
[warn] /Users/jgrgurica/Development/FOOS/alpakka/google-cloud-storage/src/main/scala/akka/stream/alpakka/googlecloud/storage/FailedUpload.scala:23:65: object JavaConverters in package collection is deprecated (since 2.13.0): Use `scala.jdk.CollectionConverters` instead
[warn]   def create(reasons: java.util.List[Throwable]) = FailedUpload(reasons.asScala.toList)
[warn]                                                                 ^
[warn] /Users/jgrgurica/Development/FOOS/alpakka/google-cloud-storage/src/main/scala/akka/stream/alpakka/googlecloud/storage/impl/GCStorageStream.scala:598:27: value Stream in package scala is deprecated (since 2.13.0): Use LazyList instead of Stream
[warn]           .mapConcat(r => Stream.continually(r))
[warn]                           ^
[warn] three warnings found
```

Can you advise me how to deal with these warnings since `scala.jdk.CollenctionConverters` and `scala.collection.immutable.LazyList` doesn't exist in previous versions of Scala?

Reply:
We will have to deal with warnings for now. It will be possible to get rid of those when a new scala-collection-compat release is out with https://github.com/scala/scala-collection-compat/pull/217